### ISSUE TITLE
Random exception

### ISSUE THREAD

**Issue by jwdeveloper** [isaackogan/TikTokLive#189]
### Describe the bug

`...`

### Expected Behaviour

`...`

### Actual Behaviour

```
Traceback (most recent call last):
  File "C:\Users\ja\PycharmProjectsss\TiktokTest\.venv\Lib\site-packages\websockets\legacy\protocol.py", line 1301, in close_connection
    await self.transfer_data_task
  File "C:\Users\ja\PycharmProjectsss\TiktokTest\.venv\Lib\site-packages\websockets\legacy\protocol.py", line 963, in transfer_data
    message = await self.read_message()
              ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\ja\PycharmProjectsss\TiktokTest\.venv\Lib\site-packages\websockets\legacy\protocol.py", line 1033, in read_message
    frame = await self.read_data_frame(max_size=self.max_size)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\ja\PycharmProjectsss\TiktokTest\.venv\Lib\site-packages\websockets\legacy\protocol.py", line 1108, in read_data_frame
    frame = await self.read_frame(max_size)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\ja\PycharmProjectsss\TiktokTest\.venv\Lib\site-packages\websockets\legacy\protocol.py", line 1165, in read_frame
    frame = await Frame.read(
            ^^^^^^^^^^^^^^^^^
  File "C:\Users\ja\PycharmProjectsss\TiktokTest\.venv\Lib\site-packages\websockets\legacy\framing.py", line 68, in read
    data = await reader(2)
           ^^^^^^^^^^^^^^^
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.11_3.11.2288.0_x64__qbz5n2kfra8p0\Lib\asyncio\streams.py", line 750, in readexactly
    await self._wait_for_data('readexactly')
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.11_3.11.2288.0_x64__qbz5n2kfra8p0\Lib\asyncio\streams.py", line 543, in _wait_for_data
    await self._waiter
asyncio.exceptions.CancelledError

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Users\ja\PycharmProjectsss\TiktokTest\main.py", line 27, in <module>
    client.run()
  File "C:\Users\ja\PycharmProjectsss\TiktokTest\.venv\Lib\site-packages\TikTokLive\client\client.py", line 171, in run
    return self._asyncio_loop.run_until_complete(self.connect(**kwargs))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.11_3.11.2288.0_x64__qbz5n2kfra8p0\Lib\asyncio\base_events.py", line 654, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "C:\Users\ja\PycharmProjectsss\TiktokTest\.venv\Lib\site-packages\TikTokLive\client\client.py", line 156, in connect
    await task
  File "C:\Users\ja\PycharmProjectsss\TiktokTest\.venv\Lib\site-packages\TikTokLive\client\client.py", line 204, in _client_loop
    async for event in self._ws_loop(initial_response):
  File "C:\Users\ja\PycharmProjectsss\TiktokTest\.venv\Lib\site-packages\TikTokLive\client\client.py", line 228, in _ws_loop
    async for response_message in self._ws.connect(*self._build_connect_info(initial_response)):
  File "C:\Users\ja\PycharmProjectsss\TiktokTest\.venv\Lib\site-packages\TikTokLive\client\ws\ws_client.py", line 145, in connect
    async for webcast_message in self.connect_loop(uri, headers):
  File "C:\Users\ja\PycharmProjectsss\TiktokTest\.venv\Lib\site-packages\TikTokLive\client\ws\ws_client.py", line 181, in connect_loop
    async for message in websocket:
  File "C:\Users\ja\PycharmProjectsss\TiktokTest\.venv\Lib\site-packages\websockets\legacy\protocol.py", line 498, in __aiter__
    yield await self.recv()
          ^^^^^^^^^^^^^^^^^
  File "C:\Users\ja\PycharmProjectsss\TiktokTest\.venv\Lib\site-packages\websockets\legacy\protocol.py", line 568, in recv
    await self.ensure_open()
  File "C:\Users\ja\PycharmProjectsss\TiktokTest\.venv\Lib\site-packages\websockets\legacy\protocol.py", line 948, in ensure_open
    raise self.connection_closed_exc()
websockets.exceptions.ConnectionClosedError: sent 1011 (internal error) keepalive ping timeout; no close frame received
Cancelling an overlapped future failed
future: <_OverlappedFuture pending cb=[_ProactorReadPipeTransport._loop_reading()]>
Traceback (most recent call last):
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.11_3.11.2288.0_x64__qbz5n2kfra8p0\Lib\asyncio\windows_events.py", line 71, in _cancel_overlapped
    self._ov.cancel()
OSError: [WinError 6] Nieprawidłowe dojście
Cancelling an overlapped future failed
future: <_OverlappedFuture pending cb=[_ProactorReadPipeTransport._loop_reading()]>
Traceback (most recent call last):
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.11_3.11.2288.0_x64__qbz5n2kfra8p0\Lib\asyncio\windows_events.py", line 71, in _cancel_overlapped
    self._ov.cancel()
OSError: [WinError 6] Nieprawidłowe dojście
Cancelling an overlapped future failed
future: <_OverlappedFuture pending cb=[_ProactorReadPipeTransport._loop_reading()]>
Traceback (most recent call last):
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.11_3.11.2288.0_x64__qbz5n2kfra8p0\Lib\asyncio\windows_events.py", line 71, in _cancel_overlapped
    self._ov.cancel()
OSError: [WinError 6] Nieprawidłowe dojście
```

### Package Version

My version of TikTokLive is `...`.

### Operating System

- [ ] Windows
- [ ] Linux
- [ ] MacOS
- [ ] Other (Specifically, `...`)

### Anything Else

If there's anything else, add it here.

**Comment by isaackogan** [comment]
@jwdeveloper Any more info? Idk what this is about :P

**Comment by glitchvr** [comment]
Did you figure this out? Dropped you an add om facebook! Many Thanks!

**Comment by isaackogan** [comment]
Fixed in 6.0.3!
